libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/TARGET_TY51822R3/PinNames.h

```diff
@@ -0,0 +1,178 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2013 Nordic Semiconductor
```

Comment:
Can I solve this issue on my next pull request? I will make another pull request to support exporting.